api/src/wfl/api/handlers.clj

```diff
@@ -91,9 +92,10 @@
                   (logr/infof "starting workload %s" uuid)
                   (workloads/start-workload! tx workload)))))]
     (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
-      (let [uuids (map :uuid (distinct (get-in request [:parameters :body])))]
-        (logr/infof "post-start endpoint called: uuids=%s" uuids)
-        (succeed (mapv (partial go! tx) uuids))))))
+      (let [body  (get-in request [:parameters :body])
+            {uuid :uuid :or {uuid body}} body]
+        (logr/infof "post-start endpoint called: uuid=%s" uuid)
+        (succeed (go! tx uuid))))))
```

Comment:
I'd recommend inlining `go!` for readability - previously it was used to map over uuids, now that's not requrired

Reply:
will update